docs/developer-guide/working-with-rules.md

```diff
@@ -311,6 +312,117 @@ Keep in mind that comments are technically not a part of the AST and are only at
 
 **Note:** One of the libraries adds AST node properties for comments - do not use these properties. Always use `sourceCode.getComments()` as this is the only guaranteed API for accessing comments (we will likely change how comments are handled later).
 
+### Accessing Code Paths
+
+ESLint is analyzing code paths while traversing AST. You can access that code path objects with `context.getCodePath()`.
+`context.getCodePath()` returns a `CodePath` object of the current location.
+
+> **Note:** That code path has not completed. If you want to get the completed code path, you can use events which will be described later.
```

Comment:
No, it does not raise any error.
Just on the way. e.g. "the code path object does not have `finalSegs` content."